glide.yaml

```diff
@@ -43,7 +43,7 @@ import:
 - package: k8s.io/client-go
   version: 82aa063804cf055e16e8911250f888bc216e8b61
 - package: github.com/projectcalico/typha
-  version: 2343517a2f18d4a47216ad93308e6a71ad7e9dc2
+  version: 3439c0869c74b6d803ef9c947b1bd166f75ac8a0
```

Comment:
How does the script know the typha pin already? Or does this need to be rev'ed once typha gets merged?

Reply:
Looks like it grabs it when it revs typha? https://github.com/tigera/process/blob/master/releases/update-libcalico.sh#L59-L88

Reply:
It's a fortune teller.

Reply:
🔮

Reply:
as long as the typha build succeeds, all is well.